src/common/reducers/entities.js

```diff
@@ -0,0 +1,62 @@
+import merge from 'lodash/merge';
+
+import * as ActionTypes from '../actions/repositories';
+
+// export only for testing
+export function repository(state={
+  __isSelected: false,
```

Comment:
The only reason I'm asking about camelCase is that we (used to) have best practice coding style defined as camelCase for JavaScript.
We don't enforce it with eslint either way, but right now it's mixed because of API objects have snake_case.

We could leave it as it is, and just use snake_case when it's returned as such from API, or take a chance and make it consistent if we normalize it anyway.